task/01-strings-tasks.js

```diff
@@ -174,6 +194,9 @@ function convertToUpperCase(str) {
  *   'info@gmail.com' => ['info@gmail.com']
  */
 function extractEmails(str) {
+    /* return ( '[123'  str.trim()']') */;
+    var array = str.split(";"); 
```

Comment:
In this case, you can immediately return the result without creating a variable.
Comments not related to logic can also be deleted